### ISSUE TITLE
meeting the license condition

### ISSUE THREAD

**Issue by alepinio** [tholman/github-corners#49]
I want to use one of your github corners in a web server whose code will be publicly available...do you have any ideas about how can I meet the condition required by the MIT license of this repo?

BTW, great job!

**Comment by tholman** [comment]
I'm no lawyer but you should be fine to go, here's a resource I find useful in a pinch https://tldrlegal.com/license/mit-license (although no substitute for a professional)

**Comment by alepinio** [comment]
As this is the first time for me I will be copying just a chunk of code of a repo (one of your octocats appearing in `index.html`) instead of cloning it and preserving its license, I did some additional research on meeting the MIT license condition. Although licensing is a subject explained everywhere in the internet, unexpectedly there isn't a lot of resources providing examples of how to meet a license condition in a practical situation.

I found rather educational this issue https://github.com/github/choosealicense.com/issues/242, and the mention in https://softwareengineering.stackexchange.com/questions/178486/what-exactly-does-the-condition-in-the-mit-license-imply about how Firefox deals with the licenses of the chunks of code it took from other projects.

I haven't decided yet if I will include your license in a HTML comment (just before the code of one of your octocats I will take from `index.html`) or in the license of my website's repository. I'm also planning to have a section in my website to credit all projects allowing me to actually build and serve the website.
